docs/addons/index.md

```diff
@@ -34,6 +34,7 @@ The framework currently supports the following add-ons.
 | [`OpaGatekeeperAddOn (Currently Not Supported, In Progress)`](./opa-gatekeeper.md)                            | Adds OPA Gatekeeper                                                                                                        |
 | [`PixieAddOn`](./pixie.md)                                             | Adds [Pixie](https://px.dev) to the EKS Cluster. Pixie provides auto-telemetry for requests, metrics, application profiles, and more. |
 | [`SecretsStoreAddOn`](./secrets-store.md)                              | Adds AWS Secrets Manager and Config Provider for Secret Store CSI Driver to the EKS Cluster                                           |
+| [`Snyk`](https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon)                                      | Adds the [Snyk Monitor](https://github.com/snyk/kubernetes-monitor) to the EKS Cluster                   |
```

Comment:
```suggestion
| [`Snyk`](https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon)  | Adds the [Snyk Monitor](https://github.com/snyk/kubernetes-monitor) to the EKS Cluster                   |
```
a bit less empty space if we want.

Reply:
This looks good, however, can you update the main page of your add-on (README) https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon to use remove reference to the ssp quickstart (since it is deprecated). This place:

```typescript
import * as cdk from 'aws-cdk-lib/core';
import * as ssp from '@aws-quickstart/ssp-amazon-eks';
import { SnykMonitorAddOn } from '@snyk-partners/snyk-monitor-eks-blueprints-addon';

const app = new cdk.App();

const addOns: Array<ssp.ClusterAddOn> = [
    new SnykMonitorAddOn({
        integrationId: '<integration ID>',
        values: {} // additional Helm chart values
    })
];

const account = '<aws account id>'
const region = '<aws region>'
const props = { env: { account, region } }

new ssp.EksBlueprint(app, { id: '<CFN stack ID>', addOns}, props)
```

Reply:
No problem @shapirov103 , it's done.